src/foam/nanos/auth/email/EmailVerificationWebAgent.java

```diff
@@ -62,7 +63,10 @@ public void execute(X x) {
       user = (User) user.fclone();
       emailToken.processToken(x, user, token);
     } catch (Throwable t) {
-      message = "Problem verifying your email.<br>" + t.getMessage();
+      String msg = t.getMessage();
+      ((Logger) x.get("logger")).error(msg);
```

Comment:
```suggestion
      ((Logger) x.get("logger")).error(msg, t);
```
wil generate a stack trace in the log file.

Reply:
@jlhughes do you want me to revert this then?